src/main/cpp/t04_swap.cpp

```diff
@@ -25,6 +25,25 @@
 using namespace std;
 
 int t04_swap() {
-    
+    int n,m,x,y,z;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'z' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4568488)